added the Strings and localisation info under technical documents (#16)
Babylonpartners/ios-playbook: TechnicalDocuments/Lokalise.md

## Lokalise Service

Lokalise is used for managing texts and translations in our apps.

Strings are edited through the [Lokalise app](https://lokalise.co). 

> Login using your github account and request access to the relevant projects.

Reviewer: ```suggestion
Create an account using your `@babylonhealth.com` email address and request access to the relevant projects.
```

## Lokalise Service

> Lokalise is used for managing texts and translations used in our apps.

Reviewer 2: ```suggestion
Lokalise is used for managing texts and translations in our apps.
```

> 

Reviewer 2: ```suggestion
```

> Strings are edited through the lokalise app [https://lokalise.co/](https://lokalise.co/). 

Reviewer 2: ```suggestion
Strings are edited through the [Lokalise app](https://lokalise.co). 
```

> Login using your github account and request acces to releavant projects.

Reviewer 2: ```suggestion
Login using your github account and request access to the relevant projects.
```

To update strings in the app the following fast-lane lanes can be used:
```
bundle exec fastlane lokalise_pull         // download all projects
bundle exec fastlane lokalise_pull_babylon     //download the main Babylon

// these download the other apps projects - they are subsets that populates the files `TargetSpecificLocalizable`
bundle exec fastlane lokalise_pull_telus
bundle exec fastlane lokalise_pull_nhs111
bundle exec fastlane lokalise_pull_bupa```
```

> To add/modify keys/values, when you are in a Project, you can use `⌘K` or click the blue `Add key`…

Reviewer 2: ```suggestion
To add/modify keys/values, when you are in a `Project` on the website, you can use `⌘K` or click the blue button `Add key`…
```
